docs/source/release/v5.1.0/indirect_geometry.rst

```diff
@@ -52,4 +52,16 @@ Bug Fixes
 - Fixed errors with the temperature correction in the ConvFit tab of the Indirect Data Analysis interface. These issues occurred when the function was evaluated at Q=0, where it is undefined.
 - Indirect Data Analysis F(Q) fit multiple workspaces can now load more than one spectra from each workspace.
 
+New
+###
+
+- :ref:`VesuvioAnalysis <algm-VesuvioAnalysis>` has been added as a beta method for analysing Vesuvio data.
+
+
+New
+###
+
+- :ref:`VesuvioAnalysis <algm-VesuvioAnalysis>` has been added as a beta method for analysing Vesuvio data.
```

Comment:
This seems to have been added twice to the release notes, please remove one.

Reply:
not sure how that happened